From 3ffd3b7678f9095b2ce7ed06a7638210cb0d35e2 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 17 Jan 2011 15:15:46 +0000
Subject: build: Fix disabling nautilus-sendto in configure

enable_nst was never set properly if we didn't use the default
value.

https://bugzilla.gnome.org/show_bug.cgi?id=639130
---
diff --git a/configure.ac b/configure.ac
index 02aaf5f..c4f2804 100644
--- a/configure.ac
+++ b/configure.ac
@@ -99,8 +99,9 @@ PKG_CHECK_MODULES(SENDTO,
 dnl nautilus-sendto plugin
 AC_ARG_ENABLE(nautilus-sendto,
               AS_HELP_STRING([--enable-nautilus-sendto=@<:@no/yes/auto@:>@],
-                             [build nautilus-sendto plugin]), ,
-                             enable_nst=yes)
+                             [build nautilus-sendto plugin]),
+                             [enable_nst=$enableval],
+                             [enable_nst=auto])
 
 if test "x$enable_nst" != "xno"; then
    PKG_CHECK_MODULES(NST,
--
cgit v0.8.3.1
